src/components/main_page/GongZhongHao.js

```diff
@@ -1,8 +1,46 @@
 import React from "react";
+//import "../../styles/main_page/_GongZhongHao.scss";
+//import "../../styles/styles/style.css.map";
 
 export default function GongZhongHao(props) {
     return (
-        <div className="gzh-container">
+        // <div className="gzh-container">
+        // </div>
+        <div className="MainGrid">
+            <article>
+                <a href="https://mp.weixin.qq.com/s?src=11&timestamp=1607286461&ver=2750&signature=jI8d4NQ08ctBdDO7eVHA0QKKAl6dhtO8Ip3QSr0PqnjQeG-YkH0PhDAx6sK00v5eAXBEv7PC8jdH0jZ0TW8g4wTCigfITtG0jxQuUfMynI*z2L47C1lXF8GcTGrsp9SR&new=1" class="button">
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/94/East_West_sushi_01.jpg/520px-East_West_sushi_01.jpg" alt="Sample photo"></img>
+                </a>
+                <div className="MainText">
+                    <br></br>
+                    <h5>多伦多日料外卖大赏</h5>
+                    <p>什么？在家也能吃到新鲜的日料？</p>
+                    
+                </div>
+            </article>
```

Comment:
It would be better if you can create a new component to simplify the work.

Reply:
https://github.com/UT-Applicataion-Development-Association/ADA-Website/pull/28/commits/007ca88840bddfe6382bdb5237a7e1a6061a5c43